### PR TITLE
Changes to use on-demand sort for on-demand column on TLEC page

### PR DESCRIPTION
**Pull request by BjornTwachtmann**

### CODE REVIEW

File: src/Controller/FindByPid/TlecController.php [hunk 1]
```diff
@@ -52,14 +52,15 @@ public function __invoke(
...
         }
 
         $upcomingBroadcast = null;
-        $streamableEpisodes = null;
+        $onDemandEpisode = null;
         $upcomingRepeatsAndDebutsCounts = ['debuts' => 0, 'repeats' => 0];
         if ($programme->getAggregatedEpisodesCount() > 0) {
-            $streamableEpisodes = $aggregationService->findStreamableDescendantEpisodes($programme, 1);
+            $onDemandEpisodes = $aggregationService->findStreamableOnDemandEpisodes($programme, 1);
+            $onDemandEpisode = $onDemandEpisodes[0] ?? null;
```
Comment (felipeparaujo) [inline src/Controller/FindByPid/TlecController.php]: why is this not being done in pps?

Comment (BjornTwachtmann) [reply src/Controller/FindByPid/TlecController.php]: well, PPS is returning an array, as you might want multiple of these

Comment (felipeparaujo) [reply src/Controller/FindByPid/TlecController.php]: ah nevermind, that was happening before as well, you just moved it up there

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**